CourseApp/Program.cs

```diff
@@ -1,13 +1,53 @@
 ﻿namespace CourseApp
 {
     using System;
+    using static System.Math;
 
     public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
+            var x_n = 1.25;
+            var x_k = 3.25;
+            var x_ch = 0.4;
```

Comment:
Please, do not use `snake_case` for naming, use `camelCase` instead
```suggestion
            var xn = 1.25;
            var xk = 3.25;
            var xch = 0.4;
```